internal/connect/command_custom_plugin_create.go

```diff
@@ -87,7 +92,18 @@ func (c *customPluginCommand) createCustomPlugin(cmd *cobra.Command, args []stri
 		return err
 	}
 
-	pluginResp, err := c.V2Client.CreateCustomPlugin(displayName, description, documentationLink, connectorClass, connectorType, resp.GetUploadId(), sensitiveProperties)
+	createCustomPluginRequest := connectcustompluginv1.NewConnectV1CustomConnectorPlugin()
+	createCustomPluginRequest.SetDisplayName(displayName)
+	createCustomPluginRequest.SetDescription(description)
+	createCustomPluginRequest.SetDocumentationLink(documentationLink)
+	createCustomPluginRequest.SetConnectorClass(connectorClass)
+	createCustomPluginRequest.SetConnectorType(connectorType)
+	createCustomPluginRequest.SetSensitiveConfigProperties(sensitiveProperties)
+	createCustomPluginRequest.SetUploadSource(
+		connectcustompluginv1.ConnectV1UploadSourcePresignedUrlAsConnectV1CustomConnectorPluginUploadSourceOneOf(
+			connectcustompluginv1.NewConnectV1UploadSourcePresignedUrl("PRESIGNED_URL_LOCATION", resp.GetUploadId())))
```

Comment:
Can we create this with one initialization statement? Look at some other create/update commands for examples.